docs/README.md

```diff
@@ -65,6 +70,49 @@ make serve-docs
 ```
 Then open a browser to [`http://localhost:8000/`](http://localhost:8000/)
 
+-----
+
+### Serving up documents globally from a fork of the repository via GitHub
+You can also take advantage of the "Generate and Push Docs" Action via the github web interface to create an online, shareable rendering of the website.
+This is particularly useful for documentation PRs, as it allows you to share a preview of your proposed changes directly via a URL to a working website.
+To take advantage of this action, you must ensure that you have forked the repository properly, so your fork includes the gh-pages branch required for the Action to run properly:
+
+#### Creating a fork that can use the Generate and Push Docs Action
+
+ 1. Log into your GitHub account via webbrowser
+ 2. Navigate to github.com/kubestellar/kubestellar
+ 3. Select the **Forks** dropdown and click on the plus sign to create a new fork ![image](https://github.com/user-attachments/assets/c0b56897-c2c4-479f-ba16-c1edd4690946)
+ 4. In the resulting dialog select your account as the owner, pick a repository name for the fork, and _be sure to uncheck the_ **"copy the** main **branch only"** _box_ <br />
+![image](https://github.com/user-attachments/assets/c5909ddd-3bf6-44c2-9102-c07c7e1d6a05)
+
+#### If you already created a fork but only included the main branch
+You can remedy the problem by propagating the _gh-pages_ branch into your fork using _git_ commands
+ 
+#### Generating a website rendered from a branch of your fork
+1. Work on the documents in a branch of your fork of the repository, and commit the changes
+2. (If you have been working on a local copy of the files, push the changes to the fork, then log into the GitHub webpage for your fork)
+3. Switch to the Actions tab in the top menu bar of the repository page
+4. Select Generate and Push Docs from the list of Actions on the left
+5. Click on the Run Workflow button on the right ![image](https://github.com/user-attachments/assets/5d3d23be-6c8f-454e-bf2c-0c58c8894957)
+6. Select the branch you wish to render and click on the second Run Workflow Button ![image](https://github.com/user-attachments/assets/427b827d-555c-4d36-b9c8-485eda002428)
+7. If that workflow completes successfully, it will automatically call the **Pages build and deployment** workflow.
+8. After a minute or so, you should be able to preview your new version of the website at `https://${repo_owner}.github.io/${fork_name}/${branch_name}`
```

Comment:
I would describe, even if only in one brief outline-type sentence, how to know when the "Pages build and deployment" workflow finishes.

Reply:
Added, with a screenclip

Reply:
Looks good.

FYI, if you wanted to, you could go further and mention that the link to the preview can be found on the page for the **pages build and deployment** workflow.

<img width="838" alt="image" src="https://github.com/user-attachments/assets/41fe6c63-b95b-4dab-8120-a02345e71d3b">